model_compression_toolkit/core/keras/keras_implementation.py

```diff
@@ -417,8 +418,8 @@ def count_node_for_mixed_precision_interest_points(self, node: BaseNode) -> bool
             node_type_name = node.framework_attr[keras_constants.ACTIVATION]
             if node_type_name in [keras_constants.SOFTMAX, keras_constants.SIGMOID]:
                 return True
-        elif node.type in [tf.nn.softmax, tf.nn.sigmoid, Conv2D, DepthwiseConv2D, Conv2DTranspose, Dense, Concatenate,
-                           Add, tf.add]:
+        elif node.type in [tf.nn.softmax, keras.layers.Softmax, tf.nn.sigmoid, Conv2D, DepthwiseConv2D, Conv2DTranspose, Dense, Concatenate,
```

Comment:
We usually use **tf.keras.layers.Softmax**, but it's the same class as **keras.layers.Softmax**.